linux_os/guide/services/ntp/chronyd_or_ntpd_set_maxpoll/tests/chrony_no_pool_nor_servers.fail.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 # packages = chrony
-#
+# remediation = none
 # profiles = xccdf_org.ssgproject.content_profile_stig
```

Comment:
Now that you are modifying this file, I think this test is not necessarily just for stig profile

Reply:
Good point! Also, we should move away from using the `profiles` tag if the tests aren't regression tests valid only in context of the profile, for parametrize we should use `variables` keyword instead. I'll try to do something with that.